deploy/osd-token-refresher/01-token-refresher.Deployment.yaml

```diff
@@ -15,6 +15,8 @@ spec:
       app.kubernetes.io/name: token-refresher
   template:
     metadata:
+      annotations:
+        last-updated: "2021-19-02"
```

Comment:
Should this be a variable itself that is updated by app-interface when the secret is?

Reply:
That sounds like a APPSRE feature to me :slightly_smiling_face:

Reply:
cc/ @maorfr thoughts? Discussion could go else where.

Reply:
This Deployment is a part of a SelectorSyncSet, which means it lands on a customer cluster. What secret in app-interface is related to that?

Reply:
From the app-interface side, if you add `qontract.recycle: "true"` annotation to the Secret itself, all using pods will be recycled.

Reply:
You can totally do that. Use the openshift templates parameter and add that parameter to the mcc saas file.

Reply:
This would be awesome, thanks @cblecker @maorfr I'm going to card this up as there will be other secrets that want the same treatment. For this PR however we'll merge as is and I'll not it in the card to be reverted once the above solution is in place.

Reply:
I'm apparently the only one who cares that this date is scrambled.